ingestion/setup.py

```diff
@@ -46,6 +46,7 @@ def get_long_description():
     "antlr4-python3-runtime==4.9.2",
     "boto3~=1.19.12",
     "botocore==1.22.12",
+    "requests-aws4auth==1.1.2",
```

Comment:
let's remove this from base requirement and move it combine it with elasticsearch plugin